nixos/doc/manual/installation/installing.xml

```diff
@@ -65,9 +66,9 @@
 
    <para>
     If you would like to continue the installation from a different machine you
-    need to activate the SSH daemon via <literal>systemctl start
-    sshd</literal>. In order to be able to login you also need to set a
-    password for <literal>root</literal> using <literal>passwd</literal>.
+    need to activate the SSH daemon via <command>systemctl start
+    sshd</command>. You then must set a password for either <literal>root</literal> or
```

Comment:
What do you think about simplifying it and dropping the "or root" bit? It'll be simpler and clearer for the user, who won't have to choose which is better.

Reply:
Well I'm thinking, someone tries to login as root and discovers "well I guess I have to set a password for root to if I want to use it?"